src/main/java/virtual_pet/VirtualPet.java

```diff
@@ -0,0 +1,73 @@
+package virtual_pet;
+
+
+public class VirtualPet {
+    private String catName;
+    private int hunger;
+    private int thirst;
+    private int attention;
+    private int bathroom;
+    private int sleepy;
+
+
+    public VirtualPet(String c) {
+        catName = c;
+        hunger = 1;
+        thirst = 1;
+        attention = 1;
+        sleepy = 1;
+        bathroom = 1;
+    }
+
+    public void changeHunger(int h) {
+        hunger += h;
+    }
+
+    public void changeThirst(int t) {
+        thirst += t;
+    }
+
+    public void changeAttention(int a) {
+        attention += a;
+    }
+
+    public void changeBathroom(int b) {
```

Comment:
When you pass the value of bathroom into the constructor and make that variable private, it belongs to the class. Therefore, you do not have to provide a parameter to changeBathroom(). Your method function would like like the following:

public void changeBathroom(){
this.bathroom += 2;
}